scripts/package/deb-after-install.sh

```diff
@@ -0,0 +1,109 @@
+#! /bin/bash
```

Comment:
I wonder if we could safely refactor some of this out, it's really similar to what we have for the RPM packaging. @phrawzty do you have any thoughts on this?

Seeing how long this is I wonder if we're doing too much in ```after-install```, does it seem abnormal for a package install to be doing this much setup? Maybe we can break this out to a cross-distro post-config script?